uwblueprint/community-fridge-kw
# feat: create and update check in endpoints

## Brief description. What is this change? 
### [Create and Update CheckIn endpoint](https://www.notion.so/uwblueprintexecs/Create-and-Update-CheckIn-endpoint-e8a75397346b45708187a59e5803878c)


## Implementation description. How did you make this change?
The service for the check in model was created along with API endpoints for create (POST) and update (PUT). Check in validators and routes (`/checkin` and `/checkin/:id`) were created.


## Steps to test
1. Create check ins that are multi-day as well as single day via Postman. For example: 
<img width="631" alt="image" src="https://user-images.githubusercontent.com/16995909/155260024-611f4f64-b605-47b4-9825-31cf68daa4f5.png">

2. Update check ins by assigning them volunteers via `volunteerID`, changing the admin status via `isAdmin`, and changing the notes via `notes`. For example:
<img width="628" alt="image" src="https://user-images.githubusercontent.com/16995909/155260198-c1dc0925-e157-455d-b5be-8c55df6c169d.png">

## Checklist
